searx/static/themes/simple/src/less/search.less

```diff
@@ -378,3 +378,12 @@ html.no-js #clear_search.hide_if_nojs {
 #categories_container {
   position: relative;
 }
+
+.favicon img {
+  height: 1.8rem;
+  width: 1.8rem;
+  border-radius: 20%;
+  background-color: #ddd;
```

Comment:
Did have any special reason to set the background color? Some favicons have a transparency; I personally would prefer to leave the background of the <img> transparent so that the background of the theme shines through the favicon. We already have a border that provides a visual demarcation.  In your opinion, would there be anything against leaving the background of the <img> transparent and deleting the background-color?

Reply:
Yeah, makes sense .. thanks for clarify :+1: